engine/src/main/java/org/terasology/engine/rendering/primitives/ChunkMesh.java

```diff
@@ -221,11 +221,7 @@ private void renderVbo(int id) {
      * @param chunkIsAnimated a boolean: true if the chunk is animated, false otherwise
      */
     public void updateMaterial(Material chunkMaterial, Vector3fc chunkPosition, boolean chunkIsAnimated) {
-        chunkMaterial.setFloat3("chunkPositionWorld",
-                chunkPosition.x() * Chunks.SIZE_X,
-                chunkPosition.y() * Chunks.SIZE_Y,
-                chunkPosition.z() * Chunks.SIZE_Z,
-                true);
```

Comment:
Yes. In https://github.com/Terasology/CoreRendering/pull/37, it uses `RenerableChunk.getRenderPosition`, which returns a value in ordinary world coordinates (to accommodate chunk-like entities which may not be aligned to the chunk grid) rather than `CoreChunk.getPosition`, which returns a value in chunk coordinates.